content/02.introduction.md

```diff
@@ -156,14 +156,50 @@ This excessive clotting in lung capillaries has been suggested to related to a d
-[@doi:10.1001/jamapediatrs.2020.1467] reviewed studies on children infected with COVID-19 during the early months of COVID-19 in China.
-Most of the children reviewed had mild symptoms such as fever, dry cough, fatigue, nasal congestion and/or runny nose or were asymptomatic.
-Gastrointestinal symptoms such as vomiting or diarrhea were occasionally reported.
-Radiologic findings were not always mentioned, but when mentioned they included bronchial thickening, ground-glass opacities, and/or inflammatory lesions.
-The results of COVID-19 infection can vary greatly among pediatric patients as well as adults.
-In particular, some children have experienced a severe inflammatory syndrome after COVID-19 infection, similar in some respects to Kawasaki disease or to Kawasaki disease shock syndrome [@doi:10.1093/jpids/piaa069;@doi:10.1001/jama.2020.10369; @doi:10.1016/j.ajem.2020.05.117].
-Heart failure can also occur [@doi:10.1161/CIRCULATIONAHA.120.048360].
+#### Subpopulations of Special Concern  
+
+In the context of the United States, persons diagnosed with COVID-19 are more likely to require hospitalization if they are of male sex, of older age, and/or of Black/African American background [@doi:10.1377/hlthaff.2020.00598; @doi:10.15585/mmwr.mm6925e1].
+African Americans have also been reported to have disproportionate risk of kidney complications from COVID-19 [@doi:10.1016/j.kint.2020.05.006].
+In addition to African Americans, disproportionate harm and mortality from COVID-19 has also been noted in Latino/Hispanic communities and in Native American / Alaskan Native communities such as the Navajo nation [@doi:10.1001/jama.2020.8598; @doi:10.1136/bmj.m1483; @doi:10.1111/jrh.12451; https://www.nytimes.com/2020/04/09/us/coronavirus-navajo-nation.html?searchResultPosition=10; @url:https://www.nytimes.com/interactive/2020/07/05/us/coronavirus-latinos-african-americans-cdc-data.html; @doi:10.1093/tbm/ibaa055].
+In Brazil, where the pandemic has also been severe, indigenous communities are likewise at special risk [@doi:10.1126/science.abc0073].
+The sizable racial disparities observed may be due to a number of factors, including different levels of pre-existing co-morbidities, such as hypertension, diabetes or lung diseases.
+They may also be influenced by the disproportionate socioeconomic burdens placed on many people of color, which can correspond to greater economic difficulties, more hazardous or crowded work or living conditions, or reduced access to health care [@doi:10.1377/hlthaff.2020.00598; @doi:10.1016/j.kint.2020.05.006; @doi:10.1001/jamanetworkopen.2020.12403].
+This might cause infections to be less likely to be diagnosed unless or until they were very severe; in the the sample studied by [@doi: 10.1377/hlthaff.2020.00598], African Americans were more likely to be diagnosed in hospital, while other groups were more likely to have been diagnosed in ambulatory settings in the community.
```

Comment:
```suggestion
This might cause infections to be less likely to be diagnosed unless or until they were very severe; in the the sample studied by [@doi:10.1377/hlthaff.2020.00598], African Americans were more likely to be diagnosed in hospital, while other groups were more likely to have been diagnosed in ambulatory settings in the community.
```